README.rst

```diff
@@ -1,11 +1,14 @@
 .. -*- mode: rst -*-
 
-|Travis|_ |Azure|_ |Circle|_ |Codecov|_ |PyPI|_ |conda-forge|_ |Zenodo|_
+|GH-Linux|_ |GH-macOS|_ |Azure|_ |Circle|_ |Codecov|_ |PyPI|_ |conda-forge|_ |Zenodo|_
 
 |MNE|_
 
-.. |Travis| image:: https://api.travis-ci.org/mne-tools/mne-python.svg?branch=master
-.. _Travis: https://travis-ci.org/mne-tools/mne-python/branches
+.. |GH-Linux| image:: https://github.com/mne-tools/mne-python/workflows/linux%20/%20conda/badge.svg?branch=master
+.. _GH-Linux: https://github.com/mne-tools/mne-python/actions?query=branch:master+event:push
+
+.. |GH-macOS| image:: https://github.com/mne-tools/mne-python/workflows/macos%20/%20conda/badge.svg?branch=master
+.. _GH-macOS: https://github.com/mne-tools/mne-python/actions?query=branch:master+event:push
```

Comment:
Preview version:

https://github.com/mne-tools/mne-python/blob/69045051f1d4a007bbc37d8159aba024e2347587/README.rst

I could add `linux / pip` etc. but since our official instructions are conda I think it makes sense to just to have the conda ones listed here.